scripts/start/test_runs.R

```diff
@@ -20,41 +20,32 @@ source("scripts/start_functions.R")
 # Source default cfg. This loads the object "cfg" in R environment
 source("config/default.cfg")
 
+#download default input data
+download_and_update(cfg)
+
 # create additional information to describe the runs
-cfg$info$flag <- "TEST" # choose a meaningful flag.
-cfg$info$user <- Sys.info()[["user"]] # Grab user name
+cfg$info$flag <- "WT" # Weekly Test (WT)
```

Comment:
```suggestion
cfg$info$flag <- "weeklyTests" 
```
I think there is no need to have it too short. Better to have something self-explaining